common/lib/xmodule/xmodule/tabs.py

```diff
@@ -411,6 +491,14 @@ def check(actual_dict, raise_error=True):
     return check
 
 
+def link_reverse_func(reverse_name):
+    """
+    Returns a function that takes in a course and reverse_url_func,
+    and calls the reverse_url_func with the given reverse_name and course' ID.
```

Comment:
Typo: course's.

Could you also please document why this workaround is needed?